zipkin-simple.js

```diff
@@ -127,6 +127,10 @@ function sendTrace (trace, data) {
 			timestamp: time,
 			value: data.annotations[i]
 		})
+
+		if (data.annotations[i] === "cr") {
+			body.duration = time - trace.timestamp
```

Comment:
so, if the goal is to add duration when you "own" the span, the same applies to timestamp.

otherwise, the same span id will store two timestamps.. one for "cs" and one for "sr". make sense?